audplot/core/api.py

```diff
@@ -8,6 +8,67 @@
 import audmetric
 
 
+def cepstrum(
+        cc_matrix: np.ndarray,
+        dur: float,
+        *,
+        channel: int = 0,
+        num_ticks: int = 10,
+        ax: plt.Axes = None,
+        cmap: str = 'jet',
+):
+    r"""Cepstrum.
+
+    Args:
+        cc_matrix: matrix with magnitude values
+        dur: duration in seconds
+        channel: channel index
+        num_ticks: number of ticks on x axis
+        ax: axes in which to draw the plot
+        cmap: color map
+
+    Example:
+        .. plot::
+            :context: reset
+            :include-source: false
+
+            from audplot import cepstrum
+
+        .. plot::
+            :context: close-figs
+
+            >>> import librosa
+            >>> x, sr = librosa.load(librosa.ex('trumpet'))
+            >>> dur = len(x) / sr
+            >>> y = librosa.feature.mfcc(x, sr)
+            >>> cepstrum(y, dur)
+
+    """
+
+    ax = ax or plt.gca()
+    cc_matrix = cc_matrix[channel] if cc_matrix.ndim == 3 else cc_matrix
+
+    n_cc, n_cepstra = cc_matrix.shape
+    ax.set_yticks(np.arange(n_cc) + 0.5)
+    ax.set_yticklabels(np.arange(n_cc))
+    ax.set_ylabel("Cepstral Coefficients")
+
+    ts = np.linspace(0, cc_matrix.shape[1], num_ticks)
+    ts_sec = ["{:4.2f}".format(i) for i in np.linspace(0, dur, num_ticks)]
+    ax.set_xticks(ts)
+    ax.set_xticklabels(ts_sec)
+    ax.set_xlabel("Time (sec)")
```

Comment:
I personally prefer `Time / s` or `Time / sec`, but if you think using `()` is more consistent with what others do in audEERING it's also fine with me.

Reply:
Don't have a strong opinion here, so I changed to `Time / s` and `Frequency / Hz`